__tests__/app.test.js

```diff
@@ -245,3 +246,74 @@ describe("GET /api/reviews/:review_id with a comment_count", () => {
       });
   });
 });
+
+describe("4.GET /api/reviews/:review_id", () => {
+  test("status:200, responds with a single review object", () => {
+    const review_id = 2;
+    return request(app)
+      .get(`/api/reviews/${review_id}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review).toEqual({
+          review_id: review_id,
+          title: "Jenga",
+          designer: "Leslie Scott",
+          owner: "philippaclaire9",
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          review_body: "Fiddly fun for all the family",
+          category: "dexterity",
+          created_at: "2021-01-18T10:01:41.251Z",
+          votes: 5,
+          comment_count: 3,
+        });
+      });
+  });
+  test("status:400, responds with a bad request error message when passed a bad review ID", () => {
+    return request(app)
+      .get("/api/reviews/notAniD")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request, Invalid Input");
+      });
+  });
+  test("status:404, respond with an error message when passed a valid ID number that is not found", () => {
+    return request(app)
+      .get("/api/reviews/9999")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Review not found for review_id: 9999");
+      });
+  });
+});
+
+describe("8.GET /api/reviews", () => {
+  test.only("status:200, responds with an array of objects containing review data in descending order", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews.length).toBeGreaterThan(0);
+        expect(reviews).toBeSortedBy("created_at", {
+          descending: true,
+          coerce: true,
```

Comment:
```js
          coerce: true,
```
You shouldn't co-erce the dates as they will all become NaN. You can take this option out